library/src/main/assets/google_map.html

```diff
@@ -90,6 +92,26 @@
       icon: 'http://maps.google.com/mapfiles/ms/icons/red-dot.png'
   });
 
+  if (title == "null" && snippet == "null") {
+        <!-- Title and snippet not set, don't create default InfoWindow -->
+  } else if (title != "null" && snippet == "null") {
```

Comment:
Could this block of logic be simplified to 
```
var content = document.createElement("div");
content.innerHTML = '';
if (title != "null") {
  content.innerHTML += '<div id="title"><b>' + title + '</b></div>';
}

if (snippet != "null") {
  content.innerHTML += '<div id="snippet">' + snippet + '</div>';
}

infoWindowContent[id] = content;
```

so we wouldn't have three blocks? Forgive me if my lack of understanding of how the innerHTML is built prevents us from combining the blocks.